### ISSUE TITLE
Create a Student Management Information System using the "C" programming language.

### ISSUE THREAD

**Issue by KanizoRGB** [section-engineering-education/engineering-education#6381]
### Topic Suggestion
Create a Student Management Information System using C

### Writing sample(s): 
1.https://www.section.io/engineering-education/the-complete-roadmap-to-blockchain-development/
2.https://www.section.io/engineering-education/machine-learning-libraries-in-cplusplus/

#### Proposed title of article
[Languages]Create a Student Management Information System using the "C" programming language.

#### Proposed article introduction
The demand for knowledge has been on the rise as we continue in the journey of civilization as a human race. As new technologies emerge in every sector of our lives, the number of learners continues to rise steadily. With such a high number of students enrolling to learning institutions, a suitable way of managing their data is needed.
In the olden days, keeping such records was done using pen and paper but since we are in the digital era, that tale is long gone. Nowadays we make use of digital records which not only have the ability to store a huge number of records but also minimize the errors which are prone to be made when solely done by a  human. 
To that effect, in this tutorial, you will learn how to create your own student management information system using the robust "C"  programming language.

#### Key takeaways 
1. How to implement record-keeping logic in code.
2. The use of data structures in C
3. The practical use of functions in C

#### Article quality
My article is unique in the sense that it explains what is happening in the code line by line. In addition to that, I take the readers through simple and easy steps of how I broke down the problem question.

#### References
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by WanjaMIKE** [comment]
Based on this [search](https://www.google.com/search?q=Create+a+Student+Management+Information+System+using+the+%22C%22+programming+language&oq=Create+a+Student+Management+Information+System+using+the+%22C%22+programming+language&aqs=chrome..69i57j69i61.325j0j9&sourceid=chrome&ie=UTF-8), this topic appears to be over-saturated on other blog sites and thus will not be of much value. Feel free to suggest another topic. Thanks

**Comment by KanizoRGB** [comment]
Hello, have you thought that I might bring a new twist to it? Or rather that my article will be more easily understandable than the rest?. You keep turning down my articles

**Comment by KanizoRGB** [comment]
Of course, anything can be found on the internet. It is the approach that will be different and that is why we have plagiarism checkers to look for originality.
